Makefile

```diff
@@ -0,0 +1,40 @@
+LANG		:= en
+LANG_FILE	:= $(shell test -f LANGS.md.bak && echo LANGS.md.bak || echo LANGS.md)
+LANG_DATA	:= $(shell grep "$(LANG)/" $(LANG_FILE))
+LANG_NAME	:= $(shell echo "$(LANG_DATA)" | sed 's/.*\[\(.*\)\].*/\1/')
+
+help:
+	@echo
+	@echo "Usage: make command [LANG=<language_short_code>]"
+	@echo
+	@echo "Language short codes are found in LANGS.md. ( For eg: LANG=es )"
+	@echo "Default Language is 'en'(English)."
+	@echo "Commands:"
+	@echo "  init	- Initialise and Start the server."
+	@echo "  serve	- Start honkit server locally for development."
+	@echo "  setup	- Setup the temporary language for the development."
+	@echo "  reset	- Reset the temporarily set language."
+	@echo "  check	- Check for root directory for various dependencies."
+	@echo "  help	- Display command list."
+
+check: book.json LANGS.md package.json $(shell which node) $(shell which npx)
```

Comment:
Why is the `node` dependency here?

Reply:
```
check: package.json book.json LANGS.md
ifneq ($(shell which node && echo $?), "0")
	@echo "Node.js not found. Please install/reinstall Node.js. NVM is recommended for installation(https://github.com/nvm-sh/nvm)."
endif
```
How about this one? (Yes, checking npx was unnecessary I think) It helps to avoid errors in further steps and give out a recommendation. NVM install node without the need for sudo access for running commands.